daemon/info.go

```diff
@@ -113,6 +113,25 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 	return v, nil
 }
 
+// SystemVersion returns version information about the daemon.
+func (daemon *Daemon) SystemVersion() types.Version {
```

Comment:
Why move this into the daemon if it can be handled outside? I think the function itself looks fine.